src/runtime.rs

```diff
@@ -130,9 +130,10 @@ impl RuntimeBuilder {
         mut self,
         function_name: &str,
         function_ptr: *mut c_void,
+        signature: CString,
     ) -> RuntimeBuilder {
         self.host_functions
-            .register_host_function(function_name, function_ptr);
+            .register_host_function(function_name, function_ptr, signature);
```

Comment:
WAMR has changed `register_host_function()` signature in wasm_export.h recently. The 3rd arguments has been removed.